controls/V-72841.rb

```diff
@@ -73,7 +79,7 @@
         This can allow unauthorized access to the database and through the
         database to other components of the information system."
   impact 0.5
-  
+
   tag "severity": "medium"
```

Comment:
As a note, for most of the core profiles, we have moved away from using the severity tag. This allows us to use the `impact` value dynamically and in cases where we need to change the `severity` of a control given conditions on the system we are better able to report the true state of the system.

See: https://github.com/djhaynes/inspec-profile-disa_stig-el7/blob/dev/controls/V-71861.rb

Note how we connected the `impact` value and the `describe` blocks together to take into account the dynamic state of the evaluation of the control. 

Soon we will have more control over the `pass`, `fail`, `skip` etc. state of the workflow. ( see: https://github.com/chef/inspec/issues/3017 ) For the time being we are using this method to fully capture the spirit of the controls.